### ISSUE TITLE
binance future support

### ISSUE THREAD

**Issue by MoMentum99** [c9s/bbgo#260]
First of all, thank you so much for great project!!

Do you have some plans for support for binance future API?

Thank you in advance. Good luck :)

**Comment by c9s** [comment]
yes, we have that in the plan. however, the design document should come up before we start implementing it.

**Comment by c9s** [comment]
the initial idea is to implement a separate Binance websocket stream for Binance futures and add a switch mode to the Binance exchange struct.

**Comment by c9s** [comment]
WIP PR https://github.com/c9s/bbgo/pull/299

**Comment by austin362667** [comment]
Hi there, BBGO supports Binance futures in main now. @MoMentum99 
Please enjoy it by enabling futures mode in configuration file. Thanks for the waiting.^^

**Comment by c9s** [comment]
already Added, closing.  thanks @austin362667 !
